testdata/networking/net_admin_cap_pod.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   containers:
   - name: network-pod
-    image: docker.io/bmeng/centos-network
+    image: docker.io/bmeng/centos-network@sha256:48da37205f9b43424e0983d4c5e7e07f77b7ba1504bbe35e2f264c75dcb4cd15
```

Comment:
The personal account should be replaced.

Reply:
aha. missed this one. thanks for catching this.  updated